CODEOWNERS

```diff
@@ -1,8 +1,4 @@
-# These are the default owners for the whole content of the `serverless-manager` repository. The default owners are automatically added as reviewers when you open a pull request unless different owners are specified in the file.
-* @m00g3n @pPrecel @dbadura @kwiatekus @cortey @anoipm @MichalKalke @halamix2
+# Repo owners list
 
-# All files and subdirectories in /docs
-/docs/ @mmitoraj @NHingerl @grego952 @IwonaLanger @nataliasitko
```

Comment:
IMO in `docs` we have non `.md` files that belong to technical writers (e.g. `docs/assets/modular-serverless.svg`)